generators/utils.js

```diff
@@ -250,9 +250,14 @@ function replacePlaceholders(body, generator) {
 function geti18nJson(key, generator) {
     const i18nDirectory = `${LANGUAGES_MAIN_SRC_DIR}i18n/en/`;
     const name = _.kebabCase(key.split('.')[0]);
-    let filename = `${i18nDirectory + name}.json`;
+    let filename;
     let render;
-
+    if (['activate', 'global', 'health', 'reset'].indexOf(name) !== -1) {
```

Comment:
You could use includes rather than index if as its more readable.  Also instead of checking for special names may be we should try with ejs ext when normal trial fails

Reply:
Ok, will do the change